<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ugrafo</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(52, 101, 164);
font: 75 italic 25pt &quot;Century Schoolbook L&quot;;</string>
   </property>
   <property name="text">
    <string>Ugrafo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="abrir_grafo">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>191</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Abir Grafo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="crear_grafo_aleatorio">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>191</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Crear Grafo Aleatorio</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_num_aleatorio">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>41</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
